pkg/operator/ceph/csi/util.go

```diff
@@ -104,15 +104,19 @@ func templateToDeployment(name, templatePath string, p templateParam) (*apps.Dep
 	return &ds, nil
 }
 
-func getToleration(provisioner bool) []corev1.Toleration {
+func getToleration(clientset kubernetes.Interface, provisioner bool) []corev1.Toleration {
 	// Add toleration if any
 	tolerations := []corev1.Toleration{}
 	var err error
 	tolerationsRaw := ""
 	if provisioner {
-		tolerationsRaw = os.Getenv(provisionerTolerationsEnv)
+		tolerationsRaw, err = k8sutil.GetOperatorSetting(clientset, provisionerTolerationsEnv, "")
```

Comment:
I suspect other places than CSI could benefit from that, are you planning on adding this too?

Reply:
I'm not sure I understand. 
In a follow-up PR all operator config will be configurable and we'll be reusing this.